### PR TITLE
fix: fully round corner on account dropdown

### PR DESCRIPTION
**Pull request by vm**
Screen Shot 2022-09-20 at 5 59 43 PM
Screen Shot 2022-09-20 at 5 59 50 PM

### CODE REVIEW

File: src/components/Web3Status/index.tsx [hunk 1]
```diff
@@ -34,12 +34,14 @@ import Loader from '../Loader'
...
 
+const FULL_BORDER_RADIUS = 9999
+
 const Web3StatusGeneric = styled(ButtonSecondary)`
   ${({ theme }) => theme.flexRowNoWrap}
   width: 100%;
   align-items: center;
   padding: 0.5rem;
-  border-radius: 14px;
+  border-radius: ${FULL_BORDER_RADIUS}px;
```
Comment (zzmp) [inline src/components/Web3Status/index.tsx:44]: Why can't this just be 100%?

**Review by zzmp — CHANGES_REQUESTED** [review]

File: src/components/Web3Status/index.tsx (hunk 1 above)
Comment (vm) [reply src/components/Web3Status/index.tsx:44]: then it becomes an oval

Comment (vm) [reply src/components/Web3Status/index.tsx:44]: Screen Shot 2022-09-20 at 6 18 46 PM

Comment (zzmp) [reply src/components/Web3Status/index.tsx:44]: Can you link to the SO post above the const?

**Review by zzmp — APPROVED** [review]

**Pull request merged**